.github/file-paths.yaml

```diff
@@ -0,0 +1,70 @@
+shared_sources: &shared_sources
+  - "shared/src"
+
+shared_specs: &shared_specs
+  - "shared/test"
+
+frontend_sources: &frontend_sources
+  - *shared_sources
+  - "frontend/**"
+  - "enterprise/frontend/**"
+  - "frontend/!(test)/**"
+  - "yarn.lock"
+  - "**/tsconfig*.json"
+  - "package.json"
+  - "babel.config.json"
+  - "postcss.config.js"
+  - "webpack.config.js"
+  - "webpack.static-viz.config.js"
+
+frontend_specs: &frontend_specs
+  - *shared_specs
+  - "frontend/test/!(__support__|__runner__)/**"
+  - "frontend/**.unit.*"
+  - "jest.config.js"
+
+frontend_all: &frontend_all
+  - *frontend_sources
+  - *frontend_specs
+
+backend_presto_kerberos:
+  - "**/presto_jdbc/**"
+  - "**/presto_jdbc.clj"
+
+backend_sources: &backend_sources
+  - *shared_sources
+  - "**deps.edn"
```

Comment:
Why is this `**deps.edn` here and not just `deps.edn`?

Reply:
@qnkhuat suggested that just in case we have some nested `deps.edn` (now or in the future)

Reply:
Hmm. I'm struggling to imagine how a `deps.edn` that is not in the other backend_sources directories can change the behaviour of the backend_sources code. @qnkhuat what scenario did you have in mind?

Reply:
I think if we include **deps.edn we'll safe ourselves from having multiple deps.edn entries.

Since most source code and `deps.edn` are not at the same level. Our normal structure is:
- top-level/deps.edn
- top-level/src/module
- top-level/test/module

So if you want to make sure both deps and src of this is covered, we will need to include 2 entries:
- top-level/deps.edn
- top-levle/src


And we have do this for any src folder.